src/modules/add.test.js

```diff
@@ -0,0 +1,31 @@
+import add from './add.js';
+import remove from './remove.js';
+
+describe('add test', () => {
+  test('adding first item', () => {
+    expect(add('Emre Book', [])).toEqual([{ description: 'Emre Book', completed: false, index: 1 }]);
+  });
+  test('adding fourth item', () => {
+    expect(add('Sajeel Book', [{ description: 'Emre Book', completed: false, index: 1 },
+      { description: 'Hello', completed: false, index: 2 },
+      { description: 'Microverse', completed: false, index: 3 }]))
+      .toEqual([{ description: 'Emre Book', completed: false, index: 1 },
+        { description: 'Hello', completed: false, index: 2 },
+        { description: 'Microverse', completed: false, index: 3 },
+        { description: 'Sajeel Book', completed: false, index: 4 }]);
+  });
+  test('adding second item', () => {
+    expect(add('Sajeel Book', [{ description: 'Emre Book', completed: false, index: 1 }]))
+      .toEqual([{ description: 'Emre Book', completed: false, index: 1 },
+        { description: 'Sajeel Book', completed: false, index: 2 }]);
+  });
+});
+
+describe('removing item', () => {
+  test('removing an item', () => {
+    document.body.innerHTML = '<div id="listContainer"><div id="1"></div><div id="2"></div></div>';
+    const list = document.querySelectorAll('.listContainer');
+    remove('<div id="1"></div>', [{ description: 'Emre Book', completed: false, index: 1 }, { description: 'Sajeel Book', completed: false, index: 2 }]);
+    expect(list).toHaveLength(1);
+  });
```

Comment:
- [ ] The test for removing an item failed, Please edit this test by mocking HTML to test if the delete function removes exactly one task element from the DOM.

![image](https://user-images.githubusercontent.com/68796232/194094152-4048d7cd-6bcc-4d23-a893-243917447b8b.png)